R/derive_advs_params.R

```diff
@@ -255,6 +255,15 @@ compute_map <- function(diabp, sysbp, hr = NULL) {
 #'
 #'   Permitted Values: character value
 #'
+#' @param constant_height HEIGHT parameter is constant
```

Comment:
I think we always want to derive BMI/BSA for all visits.

However, I wonder if we need this argument at all. Providing `constant_by_vars` should be sufficient. If it is not `NULL`, `constant_parameters` and `constant_by_vars` is specified in the `derive_param_computed()` call. Otherwise, these are not specified.

Reply:
@bundfussr I like the idea of using `constant_by_vars`.  This is updated now.  Thanks!

Reply:
I like this idea also 😄